DBZ-4499 Enable Oracle connector to execute INITIAL_ONLY snapshot.mode (#3085)
debezium/debezium: documentation/modules/ROOT/pages/connectors/oracle.adoc

|Specifies the mode that the connector uses to take snapshots of a captured table.
You can set the following values:

> `initial`:: The snapshot includes the structure and data of a captured table.

Reviewer: ```suggestion
`initial`:: The snapshot includes the structure and data of the captured tables.
```
We shouldn't speak in terms of a singular table here because that's not accurate.  We should retain the plural form.

Specify this value to populate topics with a complete representation of the data from the captured tables.

> `initial_only`:: The snapshot includes the structure and data of captured tables.

Reviewer: ```suggestion
`initial_only`:: The snapshot includes the structure and data of the captured tables.
```

Database history topics require infinite retention.
Note this mode is only safe to be used when it is guaranteed that no schema changes happened since the point in time the connector was shut down before and the point in time the snapshot is taken.

> After the snapshot is complete, the connector continues to read change events from the database's redo logs except in case of initial_only snapshot mode.

Reviewer: ```suggestion
After the snapshot is complete, the connector continues to read change events from the database's redo logs except when `snapshot.mode` is configured as `initial_only`.
```
